apis/kueue/v1beta1/workload_types.go

```diff
@@ -59,6 +59,20 @@ type WorkloadSpec struct {
 	// +kubebuilder:default=""
 	// +kubebuilder:validation:Enum=kueue.x-k8s.io/workloadpriorityclass;scheduling.k8s.io/priorityclass;""
 	PriorityClassSource string `json:"priorityClassSource,omitempty"`
+
+	// QueueingPolicy that will determine if a job can be queued.
```

Comment:
```
// QueueingPolicy determines if a workload can be admitted into a queue.
// Changing the QueueingPolicy to Never does not evict or suspend workloads that are already running.
// Possible values are:
//
//   - Never: indicates that a workload should never be admitted  
//   - Always: indicates that a workload can be evaluated for admission into it's respective queue.
//
// Defaults to Always
// +kubebuilder:validation:Enum={Never, Always}
// +kubebuilder:default="Always"
```

Reply:
just throwing another option:

`queueingPolicy: Hold | Admit`, with potential extension to `AdmitOnce` if we ever want to implement it.

Which one do you think is more understandable at first glance?

Reply:
@tenzen-y wdyt?

Reply:
`Admit` sounds good. However, I think `Hold` seems to have "keep" semantics. So if selecting `Admit`, `Suspend` might be better since `Suspend` has "temporary" semantics.

Hold: https://www.oxfordlearnersdictionaries.com/definition/english/hold_1
Suspend: https://www.oxfordlearnersdictionaries.com/definition/english/suspend?q=suspend

Also, I think we can define {Never, Always, Once}, as an alternative.

Reply:
The way I see it, we are holding the workload from being queued. It's not temporary while the configuration is set.

Reply:
I see. Let me summarize "Suspend" and "Hold" semantics.

Suspend: The job is enqueued and temporarily stopped until dequeued.
Hold: The job isn't enqueued and keeps stopped until enqueued.

If the above my understanding is correct, queuingPolicy={Hold, Admit, AdmitOnce} looks good.

Reply:
ping @alculquicondor :)

Reply:
I still like `Hold` and `Admit` better.

Reply:
+1 on `Hold` and `Admit`.
`Never` sounds like eternally queueing regardless of whether doing any operations.

Reply:
@yaroslava-serdiuk commented the following: 

> We would like to have a "stop" command in Kueue CLI (https://github.com/kubernetes-sigs/kueue/pull/1325) for workload (related to this PR) and clusterqueue(https://github.com/kubernetes-sigs/kueue/pull/1288), so from user point of view it would be really nice to have the similar naming.

Maybe having values Active/Stopped is an option?